resources/puppetlabs/lein-ezbake/template/global/ext/redhat/ezbake.service.erb

```diff
@@ -10,7 +10,12 @@
 #
 [Unit]
 Description=<%= EZBake::Config[:project] %> Service
-After=syslog.target network.target
+
+<% if EZBake::Config[:build_type] == 'foss' -%>
```

Comment:
I wonder if this actually needs a conditional or if it could just be the 'pe' line?